src/index.html

```diff
@@ -1,12 +1,375 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo_Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item ">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-content">
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Mac image"
+        >
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </div>
```

Comment:
Need to use the semantic tag for text everywhere
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina
          <br>
          5K Monoblock (MNED2UA/A)
        </h2>
```